db/migrate/20240309165235_create_contact_media.rb

```diff
@@ -0,0 +1,11 @@
+class CreateContactMedia < ActiveRecord::Migration[7.0]
+  def change
+    create_table :contact_media do |t|
+      t.string :rank
```

Comment:
@emmvs  @xxjmsxx  Correct me if I'm wrong, but shouldn't this be an Int?

Reply:
same here 😈